schema/vod-source.yaml

```diff
@@ -159,3 +161,48 @@ $defs:
           Additional information: Semicolon-separated series of additional
           information fields.
     required: ['genome_assembly', 'chrom', 'pos', 'ref', 'alt']
+  
+  LocationDescriptor:
+    description: >-
+      This descriptor is intended to reference :ref:`VRSLocation` value objects. In addition to the
```

Comment:
:ref:`VRSLocation` seems to be made up, here. If this is intended to reference a specific section of the VRSATILE docs, please tag appropriately. I think you'll find that some of this text is redundant with the build tooling when you run it.

Reply:
@ahwagner  can you help me out how to so this?